f/connectors/timelapse/timelapse.script.lock

```diff
@@ -1,2 +1,19 @@
 pandas==2.2.3
-psycopg2-binary==2.9.10
\ No newline at end of file
+psycopg2-binary==2.9.10
+numpy==1.26.4
+pytz==2024.1
+python-dateutil==2.8.2
+six==1.16.0
+tzdata==2024.1
```

Comment:
These were added because I hadn't run Timelapse in a Windmill runtime environment until now, and these were required.